app/webpacker/components/Results/ResultsFilter.jsx

```diff
@@ -60,7 +60,7 @@ export default function ResultsFilter({
         <Form.Field>
           <RegionSelector
             region={regionIso2}
-            onRegionChange={(r) => setRegion(getRegionIdWithFallback(r))}
+            onRegionChange={(e, { value }) => setRegion(getRegionIdWithFallback(value))}
```

Comment:
This can... maybe...? benefit from `useInputState`, although I'm not sure what `getRegionIdWithFallback` does. Is it compatible / can it be replaced with the default value of your refactored country selector?

Reply:
The results page is storing the region as the country name, instead of iso2. `getRegionIdWithFallback` translates from the iso2 to the name. (And also translates continents/world.)